frontend/src/pages/admin/Template/sections/Editor.vue

```diff
@@ -49,19 +57,27 @@
 </template>
 
 <script>
-
-import FormRow from '@/components/FormRow'
+import FormRow from '@/components/ForumRow'
```

Comment:
@PLCMercenary have you tested these latest changes work? This typo is failing the builds, and should have been picked up when testing the frontend still works after these latest changes